Proton/Sources/ObjC/include/PRTextStorage.h

```diff
@@ -25,21 +25,35 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@protocol DefaultTextFormattingProviding;
-@protocol TextStorageDelegate;
-@class Attachment;
+NS_SWIFT_NAME(DefaultTextFormattingProviding)
+@protocol PRDefaultTextFormattingProviding<NSObject>
+@optional
```

Comment:
Is there any reason you want them optional?

Reply:
ya! this need not be optional